README.md

```diff
@@ -54,6 +71,17 @@ sap_hana_ha_pacemaker_hacluster_node1_fqdn: hana-node1.test.local
 sap_hana_ha_pacemaker_hacluster_node2_fqdn: hana-node2.test.local
 sap_hana_ha_pacemaker_hacluster_node1_ip: 192.168.47.21
 sap_hana_ha_pacemaker_hacluster_node2_ip: 192.168.47.22
+sap_hana_ha_pacemaker_fencing_device_ip: 192.168.47.11
+sap_hana_ha_pacemaker_fencing_device:
```

Comment:
Another way to declare this dict, that will allow you to access the value of each key directly on the plays is this.

```
sap_hana_ha_pacemaker_fencing_device:
     myfencingdevice:
       type: "fence_ipmilan"
       user: "fencing_user"
       pwd: "Mysecretpassword"
       host_list: "host1.example.com,host2.example.com"
       host_map_list: "host1.example.com:VM001;host2.example.com:192.168.47.50"
```

This way you can access any value of each key like this:

```
- debug:
     msg: "{{ item.value.type }}"
   loop: "{{ sap_hana_ha_pacemaker_fencing_device|dict2items }}"
```

This is up to you really, so feel free to use which ever you are more confortable with.